src/Agoda.Analyzers/AgodaCustom/AG0043NoBuildServiceProvider.cs

```diff
@@ -86,9 +87,14 @@ private static void AnalyzeNode(SyntaxNodeAnalysisContext context)
 
             if (isServiceCollectionExtension || isExpressionServiceCollection)
             {
-                var diagnostic = Diagnostic.Create(Descriptor, memberAccessExpr.Name.GetLocation());
+                var diagnostic = Diagnostic.Create(Descriptor, memberAccessExpr.Name.GetLocation(), properties: _props.ToImmutableDictionary());
                 context.ReportDiagnostic(diagnostic);
             }
         }
+
+        private static Dictionary<string, string> _props = new Dictionary<string, string>()
+        {
+            { AnalyzerConstants.KEY_TECH_DEBT_IN_MINUTES, "10" }
```

Comment:
I think some of these estimates are very optimistic :)

Reply:
cute icons it is then!

have updated

need to add cute icons to my prompts